cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/index.js

```diff
@@ -0,0 +1,67 @@
+// import { render } from 'react-dom';
+
+// const App = () => (
+//     <section className="my-money-calendar">
+//      <div>hi, missy app</div>
+//     </section>
+// );
+
+// render(<App />, document.querySelector('#mmt-cost-of-credit-tool'));
+
+import * as idb from "idb";
```

Comment:
Unless you're using IndexedDB in this app, remove this.

Reply:
I am not sure.  I will comment comment it out for now.